src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/DataProviderPass.php

```diff
@@ -47,7 +47,7 @@ public function process(ContainerBuilder $container)
      */
     private function registerDataProviders(ContainerBuilder $container, string $type)
     {
-        $services = $container->findTaggedServiceIds('api_platform.'.$type.'_data_provider');
+        $services = $container->findTaggedServiceIds('api_platform.'.$type.'_data_provider', true);
```

Comment:
Can you change `'api_platform.'.$type.'_data_provider'` to `"api_platform.$type_data_provider"` (PHP 7 perf improvement)?

Reply:
Sure

Reply:
No, sorry, the `_data_provider` part gets concatanated with `$type` part so the `$type_data_provider` var doesn't exist

Reply:
`"api_platform.{$type}_data_provider"`

Reply:
Done